README.adoc

```diff
@@ -0,0 +1,500 @@
+            "Version": "2012-10-17",
+            "Statement": [
+                {
+                    "Effect": "Allow",
+                    "Principal": {
+                        "Service": "ecs-tasks.amazonaws.com"
+                    },
+                    "Action": "sts:AssumeRole"
+                }
+            ]
+        }
+    }
+}
+----
+
+Define an environment variable with the role ARN.
+
+----
+export TASK_ROLE_ARN='arn:aws:iam::665466731577:role/hazelcast-ecs-role'
+----
+
+Finally, let's attach the defined policy to the created role.
+
+----
+aws iam attach-role-policy --role-name hazelcast-ecs-role --policy-arn ${POLICY_ARN}
+----
+
+== Configure AWS CloudWatch
+
+If you want to read logs from your application, then you need to create AWS CloudWatch Group. Note that this step is not required, but highly recommended. If you skip it, you also need to remove CloudWatch entry from `task-definition.json` and then, in the further steps, you won't be able to see any logs from your application.
+
+To create CloudWatch group, execute the following command.
+
+----
+aws logs create-log-group --log-group-name /ecs/hazelcast
+----
+
+To allow ECS task to write to CloudWatch log group, you also need to create the following ECS role.
+
+----
+aws iam create-role \
+  --role-name ecs-execution-role \
+  --assume-role-policy-document file://role-policy.json
+{
+    "Role": {
+        "Path": "/",
+        "RoleName": "ecs-execution-role",
+        "RoleId": "AROAZV4HIPQ4SGDIWGYRK",
+        "Arn": "arn:aws:iam::665466731577:role/ecs-execution-role",
+        "CreateDate": "2020-06-29T10:44:01Z",
+        "AssumeRolePolicyDocument": {
+            "Version": "2012-10-17",
+            "Statement": [
+                {
+                    "Effect": "Allow",
+                    "Principal": {
+                        "Service": "ecs-tasks.amazonaws.com"
+                    },
+                    "Action": "sts:AssumeRole"
+                }
+            ]
+        }
+    }
+}
+----
+
+Export this role as an environment variable.
+
+----
+export EXECUTION_ROLE_ARN='arn:aws:iam::665466731577:role/ecs-execution-role'
+----
+
+Finally, attach `AmazonECSTaskExecutionRolePolicy` to the created role.
+
+----
+aws iam attach-role-policy \
+  --role-name ecs-execution-role \
+  --policy-arn arn:aws:iam::aws:policy/service-role/AmazonECSTaskExecutionRolePolicy
+----
+
+== Create Task Definition
+
+To create ECS task definition, you need first to update `task-definition.json` with the following values:
+
+* `TASK_ROLE_ARN` - ARN of the role created in the "Create IAM Role" step
+* `EXECUTION_ROLE_ARN` - ARN of the Role created in the "Configure AWS CloudWatch" step
+* `REGION` - Region that you use for your CloudWatch and ECS Cluster
+
+----
+sed -i "s~TASK_ROLE_ARN~${TASK_ROLE_ARN}~g" task-definition.json
+sed -i "s~EXECUTION_ROLE_ARN~${EXECUTION_ROLE_ARN}~g" task-definition.json
+sed -i "s~REGION~eu-central-1~g" task-definition.json
+----
+
```

Comment:
LGTM 👍 I have some minor comments:

Region value can be assigned to env variable as well:
`export ECS_REGION=$(aws configure get region)`

Another thing is, above sed commands does not work for MacOSX, I think we can add another secction for  MacOSX users:
```
sed -i '' "s~TASK_ROLE_ARN~${TASK_ROLE_ARN}~g" task-definition.json
sed -i '' "s~EXECUTION_ROLE_ARN~${EXECUTION_ROLE_ARN}~g" task-definition.json
sed -i '' "s~REGION~${ECS_REGION}~g" task-definition.json
```